### ISSUE TITLE
Are there legal/ethical concerns around scraping Juvenile records?

### ISSUE THREAD

**Issue by zacharylawrence** [Recidiviz/recidiviz-data#600]

**Comment by jessex** [comment]
The easier question: there does not appear to be any _broad_ legal concern around storing data for juveniles processed in an American criminal justice system, be it publicly scraped or directly ingested. There is nothing on the topic in any of the relevant nationwide standards or guidelines, such as CJIS or NIST 800-53. Publicly available data is generally "up-for-grabs" regardless of who is contained in that data, minus any constraints imposed by a terms of service. Individual state or local governments _might_ have their own laws on the books regarding special care for data belonging to juveniles, but that would surface during the MOU/data share agreement drafting with that jurisdiction.

(A number of states maintain separate Criminal Justice Information Systems and Juvenile Justice Information Systems, but I _believe_ these are specifically for data related to juveniles held in juvenile-focused facilities and programs, none of which we are currently ingesting or planning to ingest. That is, I believe that a juvenile tried as an adult would show up in a CJIS instead of a JJIS, and thus potentially be ingested by our system.)

The much harder question is around ethics. Is it ethical to store data for juveniles involves in an American criminal justice system? I can conceive of arguments for and against and I have an idea of which way I lean but I want to sleep on this and give it some more thought.

**Comment by zacharylawrence** [comment]
Thanks for sharing your preliminary thoughts on this @jessex!
<br>
> that would surface during the MOU/data share agreement drafting with that jurisdiction

Quick thought on this: Scraped data may not have gone through the MOU process and may not contain explicit birthdates/ages (eg. we could scrape juvenile records and not know it). This may be fine, but just something else worth considering.

**Comment by jessex** [comment]
>Scraped data may not have gone through the MOU process and may not contain explicit birthdates/ages (eg. we could scrape juvenile records and not know it)

That's correct. I was a bit ambiguous. I was referring specifically to direct ingest with that line. Individual governments that we sign MOU/data share agreements with for the purposes of direct ingest might have juvenile-relevant laws on the books, and we'd expect that to come up while we are signing those agreements. Which is to say, the first time we come across some rule around specific handling of data for juveniles, we can address it then (potentially in a region-specific fashion), if we have not already come up with juvenile-specific process as a result of the _ethical_ side of the question.

**Comment by jessex** [comment]
My proposal here is the following: do the same as we do for adults for juveniles' data that is intermingled with adults' data (i.e. for juveniles who have interacted with adult corrections and supervision facilities), and develop a juvenile-specific retention plan before we choose to ingest juvenile-specific facilities.

My thinking on this is two-fold:
* Regardless of our political or ethical thoughts on the booking, charging, sentencing, and so on of juveniles through the adult criminal justice apparatus, it does happen today. And if we are attempting to provide objective, true measurement and analysis of that apparatus, we should not try to unduly mask the data we are ingesting. To retain them for shorter periods of time or refuse to attempt to perform entity matching, for example, has difficult to predict ramifications in the long-term, but those ramifications all emanate from the decision to obscure the truth about who is actually being booked and incarcerated in American prisons and jails. This leads to lesser impact. "What gets measured gets managed" is one of the guiding thoughts behind this endeavor. Refusing to measure the impact of the incarceration state on juveniles means we cannot manage the problem. It does a disservice to the same people we are trying to protect.
* We are already planning to enforce a retention policy for adults who are ingested which will protect the privacy of released individuals, and I think that we should apply that same policy, whatever its exact mandates, to juveniles ingested from the same sources. Reasonable use of general retention policies and privacy measurements feels appropriate and ethical here without having to go a step further for the juvenile cohort. This will accomplish the same general goals of ensuring we don't improperly track juveniles once they are well removed from the incarceration system, while allowing us to get accurate, timely data on the scope of their incarceration into the hands of the people who can reduce or eliminate it.
